.github/workflows/build_and_push_docker_images.yaml

```diff
@@ -41,9 +44,14 @@ jobs:
           cd source
           tag=${{ github.ref }}
           # tag docker-latest-ce or docker-latest-ee
-          if git describe --tags --exact --match 'docker-latest-*'; then
+          if git describe --tags --exact --match 'docker-latest-*' 2>/dev/null; then
```

Comment:
the latest tag is on 4.4? or 5.0?

Reply:
it's on 5.0.11 now for both emqx/emqx and emqx